doc/source/installation/apple_arm.rst

```diff
@@ -22,11 +22,16 @@ To install the dependencies (mpi, p4est, trilinos and METIS) all together using
 .. code-block:: text
   :class: copy-button
 
-  sudo apt-get install libmuparser-dev
+  brew install muparser
 
 Clone the candi git repository in a folder of your choice  (e.g. ``$HOME/software/``). You can edit the ``candi.cfg`` file if you want to force the installation of the deal.II master version instead of the current stable version by setting ``DEAL_II_VERSION=master`` on line 97. Under Apple ARM, we only recommend the installation of the required libraries, namely parmetis, trilinos and p4est.
 
-To ensure that the Lethe test suite works, deal.II must be configured with p4est version 2.3.6, the current default candi version of p4est. Otherwise, application tests that include restart files will fail.
+To ensure that the Lethe test suite works, deal.II must be configured with p4est version 2.3.6, the current default candi version of p4est. Otherwise, application tests that include restart files will fail. We also recommend the use of trilinos 12.18.1, which is the version that has been tested with Lethe. To do so, change the ``candi.cfg`` file, line 103, to:
+
+..code-block:: text
+  :class: copy-button
+
+  TRILINOS_VERSION=12
```

Comment:
For me the default version of trilinos work just well, I did not have to change the trilinos version on my machine. Are you sure this is necessary?